app/assets/javascripts/angular/directives/mybooks.js.erb

```diff
@@ -0,0 +1,18 @@
+app.directive('errSrc', function() {
```

Comment:
So that a default "book guy" picture will be shown when a thumbnail failed to load. I have used this is several places, not sure if this is the best way to do it though..